index.html

```diff
@@ -1336,7 +1336,7 @@ <h3>Types</h3>
               <td>
 <code>VerifiableCredential</code> and, optionally, a more specific
 <a>verifiable credential</a> <a>type</a>. For example,<br>
-<code>"type": ["VerifiableCredential", "UniversityDegreeCredential"]</code>
+<code>"type": ["VerifiableCredential", "IdentityCredential"]</code>
```

Comment:
I made this more neutral, since our spec does not define `UniversityDegreeCredential`, the example here is misleading.

Obviously since the context is omitted, this term is expected to be defined as:

https://www.w3.org/ns/credentials/issuer-dependent#IdentityCredential

Reply:
I can revert this change if we get stuck on this attempt to improve this currently invalid example.

Reply:
`IdentityCredential` is going to confuse people given the charter in progress for the "Identity Credentials" work by Google, Apple, etc. We should avoid that terminology for two reasons:

1. It will get confused by the upcoming W3C WG that will be similarly named.
2. "Identity" is a lightning rod word that we should avoid in the specification, especially in examples.
3. Using an issuer-dependent term as the core example is not something that will achieve consensus. Continuing to raise PRs that use issuer-dependent terms in core examples will continue to be objected to.

Alternate suggestion: `ExampleCredential` ... but, honestly, we have an examples v2 context now... we should just use it for the examples in the specification.

Reply:
I think we are not a working group dedicated to the education vertical... and I find t he consistent bias towards academic use cases harmful to the standard... I hope we can find more neutral examples.

Reply:
Perhaps `ExampleAchievementCredential` which could include merit badges from scouting-like organizations; trophies/medals from athletic competitions; "top salesperson" from employer, professional org, etc.; and so on?